src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -238,8 +239,20 @@
             decideId = mPersistentIdentity.getEventsDistinctId();
         }
         mDecideMessages.setDistinctId(decideId);
+
         mMessages = getAnalyticsMessages();
 
+        if (mPersistentIdentity.isFirstLaunch(MPDbAdapter.getInstance(mContext).getDatabaseFile().exists())) {
```

Comment:
when automaticevents is toggled to off (in the UI) , do you have any concerns in which we might want to not even execute the automaticevents code? I have it that the code isn't even executed, but initially it was for safety, so i should remove that safety if so.